docs/environment-variables.md

```diff
@@ -58,6 +58,7 @@ This document outlines environment variables that can be used to customize behav
 | `SEMAPHORE_NOTIFY_DELAY`                 | `time.Duration`     | `1s`                                                                                        | Tuning Delay when notifying semaphore waiters about availability in the semaphore                                                                                                                                                                                        |
 | `WATCH_CONTROLLER_SEMAPHORE_CONFIGMAPS` | `bool` | `true` | Whether to watch the Controller's ConfigMap and semaphore ConfigMaps for run-time changes. When disabled, the Controller will only read these ConfigMaps once and will have to be manually restarted to pick up new changes. |
 | `SKIP_WORKFLOW_DURATION_ESTIMATION` | `bool` | `false` | Whether to lookup resource usage from prior workflows to estimate usage for new workflows. |
+| `WORKFLOW_ESTIMATION_DB_QUERY_TIMEOUT` | `time.Duration` | `5s` | Timeout for database queries when estimating workflow duration. Prevents workflow execution from being blocked when database is slow or locked. |
```

Comment:
Don't use `time.Duration` as go's duration syntax isn't a standard for other languages. We shouldn't use it for new things, instead specify TIMEOUT_SECONDS and take a number please.